apps/crn-frontend/src/network/groups/GroupProfile.tsx

```diff
@@ -68,7 +68,9 @@ const GroupProfile: FC<GroupProfileProps> = ({ currentTime }) => {
           About={() => (
             <About group={group} groupTeamsElementId={groupTeamsElementId} />
           )}
-          Calendar={() => <Calendar calendars={group.calendars} />}
+          Calendar={() => (
+            <Calendar calendars={group.calendars} groupType="interest" />
+          )}
```

Comment:
```suggestion
          Calendar={<Calendar calendars={group.calendars} groupType="interest" />}
```

Won't this work?

Reply:
Apparently it doesn't. It appears like this

<img width="1353" alt="Screenshot 2023-02-14 at 09 47 32" src="https://user-images.githubusercontent.com/16595804/218746625-8a9111e6-79e6-410b-b53d-89e49ea8f6a4.png">